src/Network/IHTTPResult.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Friendica\Network;
+
+use Psr\Http\Message\MessageInterface;
+
+/**
+ * Temporary class to map Friendica used variables based on PSR-7 HTTPResponse
+ */
+interface IHTTPResult
+{
+	/**
+	 * Gets the Return Code
+	 *
+	 * @return string The Return Code
+	 */
+	public function getReturnCode();
+
+	/**
+	 * Returns the Content Type
+	 *
+	 * @return string[] the Content Types
```

Comment:
The problem is that each HTTP header contains a array of values, so that's true for Content Type as well.

It's of course possible to merge all array values to one string in case of the content type, but I wasn't sure if there aren't cases where server maybe return more than one content type ..

Reply:
OK nope:
https://stackoverflow.com/questions/5809099/does-the-http-protocol-support-multiple-content-types-in-response-headers

I'll merge it to one string :-)